features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/css/custom.css

```diff
@@ -0,0 +1,552 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Hi Tharik,

This issue is fixed by the commits in : https://github.com/wso2/carbon-apimgt/pull/2906

Thanks
SajithD